common/djangoapps/labster/api/views.py

```diff
@@ -723,7 +723,19 @@ def _request(self, request, article_slug, *args, **kwargs):
         try:
             url_path = URLPath.get_by_path(article_slug, select_related=True)
         except (NoRootURL, ObjectDoesNotExist):
-            return Response({}, status=status.HTTP_404_NOT_FOUND)
+            # The slug could be localized. Get the english one if it exists
+            # AG: i18n codes could be more than 2 digits and may contain
+            # '-' symbol. But Matthias said that he'll make sure that he
```

Comment:
May be it is good idea, to track for that codes and put them in the logs from here. And then clean them  manually. What you think about it?